package.json

```diff
@@ -136,9 +136,9 @@
     "react-transition-group": "2.4.0",
     "scriptjs": "2.5.8",
     "sinon": "7.2.2",
-    "smartcharts-beta": "0.3.92",
+    "smartcharts-beta": "0.3.93",
```

Comment:
If you update any dependency please update the accompanying Trello card as a `SRP` (Special Release Procedure) and also add it to the PR's changelog / description. It will also be helpful to document/log what changes are added to this PR for future reference.